benchmarks/spiegelman_et_al_2016_gcubed/spiegelman_material.cc

```diff
@@ -362,7 +366,7 @@ namespace aspect
                           //analytic
                           if (constant_viscosity[c] == 0  && composition_viscosities[c] <= max_visc[c] && composition_viscosities[c] >= min_visc[c])
                             {
-                              const double drucker_prager_viscosity = compute_viscosity(edot_ii,pressure,c,constant_viscosity[c],false,min_visc[c],max_visc[c]);
+                              const double drucker_prager_viscosity = compute_viscosity(edot_ii,pressure,c,constant_viscosity[c],false,1,1e100);
```

Comment:
Wait, shouldnt the last two parameters only be important if the bool flag before them is set to true? In other words: Why do you need to change their values if they are not used? Also: What do the magical numbers mean? How are they different from min_visc and max_visc? If they are unimportant  and just need to be 'something small' and 'something large' then please use `std::numeric_limits<double>::min()` and `max()` instead.

Reply:
Yes, you are right. What do you think of this solution? I also added some comments to explain it.